pom.xml

```diff
@@ -64,18 +70,53 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-thymeleaf</artifactId>
 		</dependency>
-		
+
 		<dependency>
-		        <groupId>com.google.firebase</groupId>
-		        <artifactId>firebase-admin</artifactId>
-		        <version>${firebase.version}</version>
+			<groupId>com.google.firebase</groupId>
+			<artifactId>firebase-admin</artifactId>
+			<version>${firebase.version}</version>
 		</dependency>
 		<dependency>
-				<groupId>org.springframework.boot</groupId>
-				<artifactId>spring-boot-starter-tomcat</artifactId>
-				<scope>provided</scope>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-tomcat</artifactId>
+			<!--<scope>provided</scope>-->
```

Comment:
I left this commented code because I had application deploying issues. If a deleted this one, the run will fail

Reply:
so, when committing, revert the change, then you can apply again in your local